src/state/quick-open-mixin.ts

```diff
@@ -28,4 +28,9 @@ export default <T extends Constructor<HassElement>>(superClass: T) =>
         }
       });
     }
+
+    private isOSCtrlKey(e: KeyboardEvent) {
+      const isMac = /(Mac|iPhone|iPod|iPad)/i.test(navigator.platform);
+      return isMac ? e.metaKey : e.ctrlKey;
+    }
```

Comment:
mac OS has both a metaKey and a ctrlKey.

This means on mac, we'd be stealing _four_ hotkeys instead of two. Not a huge deal in this case since chrome on mac OS doesn't reserve Ctrl+P for anything, but if we can avoid taking away a shortcut option, we should.

Reply:
Not sure we should even check for this, but if we do, we should do it only once and not on every keydown.

Reply:
I'll make it a user-configurable option

Reply:
Done